camel-sap/com.sap.conn.jco.linux.x86/pom.xml

```diff
@@ -33,7 +33,7 @@
 			<plugin>
 				<groupId>org.eclipse.tycho</groupId>
 				<artifactId>tycho-maven-plugin</artifactId>
-                <version>1.2.0</version>
+                <version>1.3.0</version>
```

Comment:
it was already here before but space are used instead of tabs like other lines. it has been correctly rectified for some othe rmodifications